Automattic/wp-calypso
# Inline Help: Adds a checklist button with progress bar and notification

* Adds a checklist button with progress bar and notification to Inline Help

Our hypothesis is this will increase the number of people who start to work on their checklist.

<img width="355" alt="screen shot 2018-08-21 at 1 03 46 pm" src="https://user-images.githubusercontent.com/2124984/44417121-ae919080-a542-11e8-8415-04f747687229.png">

**Steps to test:**

* Switch to this PR
* Navigate to My Sites
* Click the Help icon
* Check for the presence of the checklist progress bar; does clicking on it bring you to the site's checklist? Does the progress match?
